bugsnag-android-core/src/main/java/com/bugsnag/android/ImmutableConfig.kt

```diff
@@ -90,13 +90,14 @@ internal fun convertToImmutableConfig(config: Configuration): ImmutableConfig {
         else -> ErrorTypes(false)
     }
 
+    @Suppress("UNNECESSARY_SAFE_CALL", "USELESS_ELVIS")
```

Comment:
I hope all changes in this file aren't required if we're using ConfigInternal everywhere internally.

Reply:
These changes have been reverted as they are no longer necessary.